lib/sharing/gists.dart

```diff
@@ -7,7 +7,6 @@ library gists;
 import 'dart:async';
 import 'dart:convert' show JSON;
 import 'dart:html';
-
```

Comment:
We want to keep a line between the different types of imports (`dart:` / `package:` / `file`).